dotCMS/src/curl-test/Content Resource.postman_collection.json

```diff
@@ -6,13 +6,97 @@
 		"schema": "https://schema.getpostman.com/json/collection/v2.1.0/collection.json"
 	},
 	"item": [
+		{
```

Comment:
This change was merged on master. Don't know if is ok to bring it to release since, probably, will create conflicts when merging release into master.

Reply:
I have to remove this change